test/Conversion/TorchToTosa/torch-backend-to-tosa-backend-pipeline-tm-scan.mlir

```diff
@@ -0,0 +1,45 @@
+// RUN: torch-mlir-opt -pass-pipeline='builtin.module(torch-backend-to-tosa-linalg-backend-pipeline)' -split-input-file -verify-diagnostics %s | FileCheck %s
```

Comment:
There's already a file for testing our pass. We can move this to that file instead of adding another one.

Reply:
Fixed